dashboard/app/views/followers/student_user_new.html.haml

```diff
@@ -80,7 +83,7 @@
                   = t('signup_form.gender')
                 = f.select :gender, gender_options
             %br/
-            = submit_tag 'Register', class: 'btn btn-primary'
+            = submit_tag 'Register', class: 'btn btn-primary', id: 'register_join_section'
```

Comment:
What does `id: 'register_join_section'` get used for?

Reply:
Leftover from the changes I refactored out of this PR to include some client-side validations on the form. Will remove